Modules/ui.js

```diff
@@ -0,0 +1,35 @@
+import { DateTime } from './luxon.min.js';
+import Store from './store.js';
+
+export default class UI {
+  static showDate = () => {
+    document.getElementById('date').innerHTML = DateTime.now();
+  }
+
+  static displayui = (book) => {
+    const tbody = document.getElementById('lists');
+    tbody.insertAdjacentHTML(
+      'beforeend',
+      `<tr> <td>${book.title} by ${book.author}</td> 
+         <td><button class="remove" type="button">Remove</button></td> </tr> `,
+    );
+  }
+
+  static displayBooks() {
+    const books = Store.getbooks();
+    books.forEach((book) => {
+      UI.displayui(book);
+    });
+  }
+
+  static deleteBook(target) {
+    if (target.classList.contains('remove')) {
+      target.parentElement.parentElement.remove();
+    }
+  }
+
+  static clearFields() {
```

Comment:
- [ ]  Please use `arrow` syntax for functions. ( 👉 ES6 syntax, Project requirements).

Reply:
@OLIPLICHE  

Thank you for the reminder. 😺